sapi/cli/php_cli.c

```diff
@@ -118,7 +118,11 @@ static const char HARDCODED_INI[] =
 	"implicit_flush=1\n"
 	"output_buffering=0\n"
 	"max_execution_time=0\n"
-	"max_input_time=-1\n";
+	"max_input_time=-1\n"
+	"register_argc_argv=1\n";
```

Comment:
This was intentionally removed, since the implementation just ignores the INI for the CLI entirely. In practice this will not hardcode the INI anyways, since the `-d` flag still takes precedence.